src/components/CommentTeaser/CommentTeaser.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { css, merge } from 'glamor'
+import colors from '../../theme/colors'
+import { serifRegular14 } from '../Typography/styles'
+import CommentHeader from '../Comment/CommentHeader'
+import CommentTeaserHeader from './CommentTeaserHeader'
+import CommentTeaserFooter from './CommentTeaserFooter'
+
+const styles = {
+  root: {
+    marginBottom: '10px',
+    '& + &': {
+      borderTop: `1px solid ${colors.divider}`,
+      paddingTop: '20px'
+    }
+  },
+  body: {
+    ...serifRegular14,
```

Comment:
I made it a habit to always set color just below each font style, unless you want to explicitly depend on the color inherited from a parent element.

Reply:
Thanks for the tip, done.